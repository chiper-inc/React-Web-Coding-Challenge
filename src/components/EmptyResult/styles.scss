.error-data {
  @apply mt-8 flex-col flex justify-center items-center ;

  .content-image{
    img{
      @apply w-full h-full
    }
    @apply w-32 h-32;
  }
  .text {
    @apply text-2xl md:text-4xl mt-4 text-red-700;
  }
}
