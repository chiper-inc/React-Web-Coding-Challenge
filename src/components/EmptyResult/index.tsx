import { noFound } from '../../assets/images';
import './styles.scss';

const EmptyResult = () => {
  return (
    <div className="error-data">
      <div className="content-img">
        <img src={noFound} alt="No found" />
      </div>
      <p className="text">No results...</p>
    </div>
  );
};

export default EmptyResult;
