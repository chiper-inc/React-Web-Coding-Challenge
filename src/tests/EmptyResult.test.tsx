import React from 'react';
import { shallow } from 'enzyme';
import EmptyResult from '../components/EmptyResult';
test('renders the component EmptyResult', () => {
  const component = shallow(<EmptyResult />);
  expect(component).toMatchSnapshot();
});
