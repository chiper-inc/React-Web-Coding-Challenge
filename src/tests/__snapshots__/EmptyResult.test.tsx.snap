// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component EmptyResult 1`] = `
<div
  className="error-data"
>
  <div
    className="content-img"
  >
    <img
      alt="No found"
      src="noFound.png"
    />
  </div>
  <p
    className="text"
  >
    No results...
  </p>
</div>
`;
